### ISSUE TITLE
what kind of multi-label classifier does you used?

### ISSUE THREAD

**Issue by ZhenhuiTang** [ChristoferNal/multi-nilm#6]
Hi,

I have read your paper and code. However, it is not clear what kind of multi-label classifier that you used in your experiment. Specifically, What kind of training loss is used with neural network as your said in paper. How to train your neural network for multi label classification task. Is that like BR or CC by using sk-learn?

Thanks.
